### ISSUE TITLE
Suggestion for instructions.txt file

### ISSUE THREAD

**Issue by ibgregr** [gaugo87/PIXEstL#9]
removed

**Comment by ibgregr** [comment]
Oh...and also update the filenames within the zip file to include the basename of the image file that was processed.  That way I can have multiple unzipped at the same time on my desktop for comparison.  So you could just prepend the image file basename to the current filenames being used.

Thanks!
Greg

**Comment by ibgregr** [comment]
I just finished doing about a weeks worth of tweaking of my palette file (using CMYK only) so that the results from your program look as close as I could get to the results for the color lithophane app on LithophaneMaker.com.  During this tweaking I found it hard to keep up with the changes I was making to each of the colors.  So, I decided to have the palette file added to the zip file so that I could look at the preview and also see the palette that was used.  This saved me a couple of times!  

So to summarize the changes I made, I added the command line parameters to the instructions.txt file and added the palette file used to the zip file.  So at any point I can alway reprocess an image using the settings used previously.
